.prettierignore

```diff
@@ -0,0 +1,12 @@
+lib
+dist
+plugins
+src-gen
+i18n
+gen-webpack*
+.browser_modules
+arduino-ide-extension/src/node/resources
+cli-protocol
+*color-theme.json
+arduino-icons.json
+.github
```

Comment:
```suggestion
```

It is useful to format the files under this folder as well.

Reply:
I removed it and formatted the workflow resources.